src/components/Categories.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
-import Header from './Header';
+import { useDispatch } from 'react-redux';
+import { checkBookStatus } from '../redux/categories/categories';
 
 export default function Categories() {
+  const dispatch = useDispatch();
+
   return (
     <div>
-      <Header />
-      <button type="button">Check status</button>
+      <button
+        type="button"
+        onClick={() => dispatch(checkBookStatus())}
+      >
+        Check status
+      </button>
```

Comment:
- [ ] I kindly suggest that you display the return statement of 'Under Construction' underneath your button. Here is an example video below:

https://user-images.githubusercontent.com/52934128/179803825-4673312f-0373-48c6-bc54-2dc15787d224.mp4